app/admin/governance/dids.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from fastapi import APIRouter, Depends
+from aries_cloudcontroller import AriesAgentControllerBase
+
+from facade import yoma_agent
+
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/admin/governance/dids", tags=["admin", "governance"])
+
+
+@router.get("/trusted-registry", tags=["did"])
+async def get_trusted_registry(
+    aries_controller: AriesAgentControllerBase = Depends(yoma_agent),
+):
+    """
+    Retrieve Trusted partner list from Trust Registry
+
+    Parameters:
+    -----------
+    api_key: Header(None)
+        The admin API key
+
+    Returns:
+    --------
+    trusted_partners: [dict/JSON]
+        List of unique trusted partners
+    """
+    all_dids_on_ledger = await aries_controller.wallet.get_dids()
```

Comment:
Is this the way we'll store trusted partners? I thought did's in a wallet are ones that belong to that entity. I guess it's nice if this can work this way.

I assumed there was a mechanism for looking up public did's on the ledger for which the yoma agent is the endorser... that was my impression of how this would work. Though it's probably better to do it by storing it in the wallet...

Reply:
This list should be generated on the fly. this returns the results that are "public" 

Not sure, maybe there is a better wy of doing it (directly from the ledger?)

Reply:
let me know if you can think of a better way

Reply:
@lohanspies please help...

Reply:
I discussed this yesterday with @lohanspies - thought this was fine. But if we can make this better, please, say @lohanspies